package.json

```
@@ -78,11 +80,18 @@
     "eslint-config-standard-with-typescript": "^43.0.0",
     "eslint-plugin-simple-import-sort": "^12.0.0",
     "eslint-plugin-unused-imports": "^3.0.0",
-    "landlubber": "^2.0.0",
+    "eslint-plugin-react": "^7.34.1",
+    "eslint-plugin-react-hooks": "^4.6.0",
+    "eslint-plugin-react-refresh": "^0.4.6",
     "prettier": "^3.0.0",
     "tsc-alias": "^1.8.2",
     "tsup": "^8.0.1",
-    "tsx": "^4.6.2",
-    "typescript": "^5.1.0"
+    "tsx": "^4.7.1",
+    "typescript": "^5.4.2",
+    "vite": "^5.1.6"
+  },
+  "peerDependencies": {
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0"
```

```suggestion
  }
```

Can you move this above deps? See https://github.com/seamapi/react/blob/main/package.json#L111


```json
  "peerDependencies": {
    "@types/react": "^16.8.0 || ^17.0.0 || ^18.0.0",
    "@types/react-dom": "^16.8.0 || ^17.0.0 || ^18.0.0",
    "react": "^16.8.0 || ^17.0.0 || ^18.0.0",
    "react-dom": "^16.8.0 || ^17.0.0 || ^18.0.0"
  },
  "peerDependenciesMeta": {
    "@seamapi/types": {
      "optional": true
    },
    "@types/react": {
      "optional": true
    },
    "@types/react-dom": {
      "optional": true
    }
  },